### ISSUE TITLE
markdown

### ISSUE THREAD

**Issue by 3052** [abhinav/doc2go#256]
is it possible to generate markdown instead of HTML - similar to this

https://github.com/chocolacula/gdmd

**Comment by abhinav** [comment]
Hey @3052, sorry, no I don't plan for doc2go to generate Markdown.
I want to keep it specialized to HTML so that it can focus on things like cross-linking from inside code snippets, search, etc.
For Markdown, gdmd as you've pointed out and [gomarkdoc](https://github.com/princjef/gomarkdoc) are good options.
I know that the latter relies on the Markdown functionality [built into go/doc itself](https://pkg.go.dev/go/doc/comment#Printer.Markdown).

**Comment by 3052** [comment]
you sure about that? `go/doc/comment` only handles documentation comments, not generating documentation for an entire package

**Comment by abhinav** [comment]
Right, go/doc/comment only provides an API to generate Markdown from comment.s
[gomarkdoc](https://github.com/princjef/gomarkdoc) uses these APIs and others to generate Markdown documentation for multiple packages.

**Comment by 3052** [comment]
jesus the go.sum is 9 times the size of even yours:

https://github.com/princjef/gomarkdoc/blob/master/go.sum

no thank you
